db/schema.rb

```diff
@@ -26,11 +26,11 @@
     t.string   "title"
     t.text     "body"
     t.string   "type"
-    t.integer  "user_id"
+    t.integer  "person_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
   end
 
-  add_index "posts", ["user_id"], name: "index_posts_on_user_id"
+  add_index "posts", ["person_id"], name: "index_posts_on_person_id"
 
 end
```

Comment:
I'm concerned about the new schema.  The change to the post STI was already merged in the models, was the schema change recorded?